cmake/cvars.cmake

```diff
@@ -43,4 +63,23 @@ add_compile_definitions(
 	CVAR_MENU_BAR_OPEN="${CVAR_MENU_BAR_OPEN}"
 	CVAR_PREFIX_CONTROLLERS="${CVAR_PREFIX_CONTROLLERS}"
 	CVAR_PREFIX_ADVANCED_RESOLUTION="${CVAR_PREFIX_ADVANCED_RESOLUTION}"
-)
\ No newline at end of file
+
+	LUS_DEFAULT_KB_MAPPING_A=${LUS_DEFAULT_KB_MAPPING_A}
```

Comment:
it looks like all the lines above this have `${}` surrounded by `""`

i'm not sure if that's required syntax for cmake or not but if these don't break as

```suggestion
	LUS_DEFAULT_KB_MAPPING_A="${LUS_DEFAULT_KB_MAPPING_A}"
```

etc i'd think we'd want to do that for consistency

Reply:
The quotes result in syntax errors. It passes the physical string to the function.

Reply:
I agree with Bria's naming suggestion here. Can we get this in?

Reply:
I'm trying it again just-in-case it does work this time, even though it won't. :P

Reply:
![image](https://github.com/user-attachments/assets/881d1127-6b6e-45b0-b751-048720157f2e)